static/app/components/replays/replayHighlight.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import CHART_PALETTE from 'sentry/constants/chartPalette';
+import {ReplayDurationAndErrors} from 'sentry/views/replays/types';
+
+import ScoreBar from '../scoreBar';
+
+interface Props {
+  data: ReplayDurationAndErrors | undefined;
+}
+
+function replayHighlight({data}: Props) {
+  let score = 1;
+
+  if (data) {
+    // Mocked data 👇 - this will change with the new backend
+    const pagesVisited = 1;
+    const {count_if_event_type_equals_error: errors, 'equation[0]': durationInSeconds} =
+      data;
+
+    const pagesVisitedOverTime = pagesVisited / (durationInSeconds || 1);
+
+    score = (errors * 25 + pagesVisited * 5 + pagesVisitedOverTime) / 10;
+    score = Math.floor(Math.min(10, Math.max(1, score)));
+  }
+
+  const palette = new Array(10).fill([CHART_PALETTE[0][0]]);
```

Comment:
is this color going to work in dark mode?

Reply:
Yes! 
![image](https://user-images.githubusercontent.com/39612839/182244568-9c962640-cdb4-4742-9274-2d03482331f6.png)

Reply:
![Uploading Screen Shot 2022-08-01 at 1.55.18 PM.png…]()